src/main/java/org/sciborgs1155/robot/flywheel/Flywheel.java

```diff
@@ -0,0 +1,88 @@
+package org.sciborgs1155.robot.flywheel;
+
+import static edu.wpi.first.units.Units.Volts;
+import static org.sciborgs1155.robot.flywheel.FlywheelConstants.*;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine.Direction;
+import java.util.function.DoubleSupplier;
+import monologue.Logged;
+import monologue.Annotations.Log;
+
+import org.sciborgs1155.robot.Robot;
+
+public class Flywheel extends SubsystemBase implements AutoCloseable, Logged {
+  private final FlywheelIO flywheel;
+  private final SysIdRoutine sysId; //sysIdoogabooga
+
+  @Log.NT private final PIDController flywheelPID = new PIDController(kP, kI, kD);
+
+  private final SimpleMotorFeedforward flywheelFeedforward = new SimpleMotorFeedforward(kS, kV, kA);
+
+  /** Creates real or simulated flywheel based on {@link Robot#isReal()} */
+  public static Flywheel create() {
+    return Robot.isReal() ? new Flywheel(new RealFlywheel()) : new Flywheel(new SimFlywheel());
+  }
+
+  public Flywheel(FlywheelIO flywheel) {
+    this.flywheel = flywheel;
+    sysId =
+        new SysIdRoutine(
+            new SysIdRoutine.Config(),
+            new SysIdRoutine.Mechanism(
+                (v) -> flywheel.setVoltage(v.in(Volts)), null, this, "flywheel"));
+
+    SmartDashboard.putData("quasistaticBack", quasistaticBack());
```

Comment:
smart dashboard puts these all to the exact same table, name them "flywheel quasistatic back" etc